cpp/src/Slice/GrammarUtil.h

```diff
@@ -214,30 +214,18 @@ class ClassIdTok : public GrammarBase
     int t;
 };
 
-}
-
-//
-// Stuff for flex and bison
-//
-
-#define YYSTYPE Slice::GrammarBasePtr
-#define YY_DECL int slice_lex(YYSTYPE* yylvalp)
-YY_DECL;
-
-//
-// I must set the initial stack depth to the maximum stack depth to
-// disable bison stack resizing. The bison stack resizing routines use
-// simple malloc/alloc/memcpy calls, which do not work for the
-// YYSTYPE, since YYSTYPE is a C++ type, with constructor, destructor,
-// assignment operator, etc.
-//
-#define YYMAXDEPTH  10000
-#define YYINITDEPTH YYMAXDEPTH // Initial depth is set to max depth, for the reasons described above.
+// ----------------------------------------------------------------------
+// TokenContext: custom type for storing the location of matched tokens.
+// ----------------------------------------------------------------------
```

Comment:
```suggestion
 // Stores the location of matched tokens.
```

Reply:
I've changed the comment to read 'stores the location of tokens.', but kept the extra //----... for consistency, since we have them on every class in the header file (we seem to like formatting them that way in the compiler code)